Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM grafana/grafana:2.6.0
-MAINTAINER Jimmi Dyson <jimmidyson@gmail.com>
+FROM grafana/grafana:latest
```

Comment:
Not a good idea, ever, to track latest as you're then at the mercy of the from image provider ensuring backwards compatibility.

4.0.2 would be great.